xds/internal/balancer/clusterresolver/clusterresolver.go

```diff
@@ -421,11 +402,7 @@ func (c *ccWrapper) ResolveNow(resolver.ResolveNowOptions) {
 }
 
 func (c *ccWrapper) NewSubConn(addrs []resolver.Address, opts balancer.NewSubConnOptions) (sc balancer.SubConn, err error) {
-	if opts.StateListener == nil {
-		// If already set, just allow updates to be sent directly to the
-		// child's listener.  Otherwise, we are responsible for forwarding the
-		// update we'll receive to the proper child.
-		opts.StateListener = func(state balancer.SubConnState) { c.b.UpdateSubConnState(sc, state) }
-	}
+	// No need to override opts.StateListener; just forward all calls to the
+	// child that created the SubConn.
```

Comment:
Optional nit: I don't think you need this comment anymore. If we have this comment we'd have to restate this in every balancer since we switched over.

Reply:
Yeah, this should have just been deleted entirely since ccWrapper embeds ClientConn.  Done.